packages/jaeger-ui/src/components/common/RelativeDate.tsx

```diff
@@ -12,22 +12,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import React from 'react';
 import dayjs from 'dayjs';
 
 import { formatRelativeDate } from '../../utils/date';
 
 type Props = {
   fullMonthName: boolean | undefined | null;
   includeTime: boolean | undefined | null;
-  value: number | Date | any;
+  value: number | Date;
 };
 
 // TODO typescript doesn't understand text or null as react nodes
 // https://github.com/Microsoft/TypeScript/issues/21699
-export default function RelativeDate(props: Props): any {
+export default function RelativeDate(props: Props): JSX.Element {
```

Comment:
why can't this be a string instead of element?

Reply:
Because RelativeDate is used as a JSX element not like a string for ex: `<RelativeDate props />` so when I use string as a return type it throw an error.